src/src/Components/Sidebar.svelte

```diff
@@ -1,18 +1,19 @@
-<!-- Sidebar.svelte -->
 <script lang="ts">
     import { createEventDispatcher, onMount, onDestroy } from 'svelte';
     export let sidebarOpen: boolean;
+    export let toggleButton;
     export let selectFranchise: (id: number) => void;
 
     const dispatch = createEventDispatcher();
+    let sidebar;
```

Comment:
This should be typed, probably with `HtmlDivElement` or `HtmlElement`.

Reply:
Yes I agree in my new pullrequest i made it "HtmlElement | null":

https://github.com/BowenGeense/MovieShowcase/pull/3/files#diff-5fa8bc1e300230c2638d059cc9cd608ab21c79c6733942d5ddb7ca00bae85311